TECHMANIA/Assets/Scripts/Paths.cs

```diff
@@ -270,4 +276,17 @@ public static string FullPathToUri(string fullPath)
             .Replace("?", "%3f")
             .Replace("@", "%40");
     }
+
+    public static string ReplacePlatformPathText(string fullPath)
```

Comment:
I would call this `HidePlatformInternalPath` so it better indicates what this method does.